### ISSUE TITLE
Add support for days and weeks in parse_duration_ns

### ISSUE THREAD

**Issue by freeseacher** [open-policy-agent/opa#2719]
##  Expected Behavior
Currently, function parse_duration_ns is only supporting times defined in  https://golang.org/pkg/time/#ParseDuration, but, actually I need to check duration in Prometheus rules. Here we can see a much bigger durations. 
https://prometheus.io/docs/prometheus/latest/querying/basics/#time-durations
Can you please add days and weeks?

**Comment by tsandall** [comment]
If we assume that days, weeks, and years are always 24h, 7d, and 365d respectively, this is a relatively easy change to make. The problem is that this is not always true--but maybe that's not important for the majority of use cases.

**Comment by freeseacher** [comment]
not stale

**Comment by anderseknert** [comment]
@freeseacher stale just means no one is working on this, and it's not currently lined up for the near future unless someone in the community steps up. It doesn't mean it's getting closed.

**Comment by 26tanishabanik** [comment]
@tsandall, `parse_duration_ns` is exactly being referred in the repo?

**Comment by sspaink** [comment]
Hey @freeseacher It has been some time, but are you still interested in seeing this implemented? Adding only days and weeks could be possible.

**Comment by freeseacher** [comment]
Yep I am still interested. At the moment we stick to hours but it is not
obvious why :)

Best Regards,
Aleksey Shirokikh
